Add a section to CONTRIBUTING.md on how Mozilla's subject-matter experts should contribute. (#358)
mozilla/standards-positions: CONTRIBUTING.md

## Discussing Mozilla's Position on a Web Specification

> ### For Mozilla's subject-matter experts

Reviewer: ```suggestion
### For Mozilla's Subject-Matter Experts
```

> ### For Mozilla's subject-matter experts

Reviewer: I would move this below the "broader community" section.

Author: Yeah, I initially wanted to do that except that I felt that "broader community" needed to be relative to some antecedent.  I guess I can just do it anyway.

You might be asked for your opinion
on specifications where you are one of the experts within the Mozilla community,
or feel like you have valuable contributions to make
to issues on other specifications even where you haven't explicitly been asked to do so.

> This is the purpose of this repository:

Reviewer 2: Maybe move this to be the first sentence and rephrase? Something like:
 
"The purpose of this repository is to help Mozilla's technical leaders come to a position about new features being added to the Web. As such, you might be asked for your opinion..."

Author: I did a variant of what you suggested.

to help the relevant people in Mozilla's technical leadership come to a conclusion.

The goal of the discussions in this repository is

> to figure out what we think about a specification.

Reviewer 2: ```suggestion
to figure out what we think about a proposal or specification.
```

The key factors to consider are:

> * how useful we think the specification is likely to be (or how important the use case it's trying to address is), relative to the likely cost of implementing it and maintaining that implementation

Reviewer 2: ```suggestion
* how useful we think the proposed feature is likely to be (or how important the use case it's trying to address is), relative to the likely cost of implementing it and maintaining that implementation
```

> * whether there are any critical problems we see in the current proposal.

Reviewer 2: ```suggestion
* whether there are any critical problems we see in the current proposal, particularly around privacy, security, and/or implementability across multiple engines.
```

Author: I don't want to push quite that hard towards those areas (since I think it's worth considering others too), so I listed them as examples rather than as "particularly".

It's useful to discuss these in the relevant issue in this repository.

At the same time,
we should avoid doing things that move

> discussions that should happen in the standards body's normal discussion mechanisms

Reviewer 2: "that move discussions that" sounds a bit odd to me... could that be rephrased in some other way?

Author: I at least found "to move discussions that", but didn't see anything better.

from happening instead in the standards-positions issue.
When it makes sense to do so,

> it's good to raise issues through the normal issue trackers for the specification.

Reviewer 2: ```suggestion
it's good to raise issues through the normal issue trackers for the proposal.
```

However, when doing so, it's worth being careful to avoid giving a *mistaken* impression
that engagement in the discussion constitutes support.
In some cases, avoiding this impression may require having some of the discussion in the standards-positions issue.
However, in many cases that can be avoided
by giving appropriate context when filing issues, such as:
"I was reading [specification] in order to understand it better
to help evaluate what Mozilla thinks of it,
and while doing that I noticed ..."

Members of the Mozilla community are
welcome to judge that we've come to sufficient consensus in the issue,
and make a pull request to document that consensus by changing `activities.json`.
When this happens, we'd like to try to keep the technical discussion
about what the position is
happening in the issue
(so that it stays in one place),
and limit the discussion in the pull request
to the details of making the change to `activities.json`.

> ### For the broader community

Reviewer: ```suggestion
### For the Broader Community
```
